Client/Frontend/Settings/AppSettingsOptions.swift

```diff
@@ -407,6 +407,24 @@ class ExportBrowserDataSetting: HiddenSetting {
     }
 }
 
+class EnableActivtyStreamSetting: HiddenSetting {
+    let profile: Profile
+
+    override init(settings: SettingsTableViewController) {
+        self.profile = settings.profile
+        super.init(settings: settings)
+    }
+
+    override var title: NSAttributedString? {
+        // Not localized for now.
+        return NSAttributedString(string: "Enable the New Tab Experience", attributes: [NSForegroundColorAttributeName: UIConstants.TableViewRowTextColor])
+    }
+
+    override func onClick(_ navigationController: UINavigationController?) {
+        FeatureSwitches.activityStream.enrollMember(profile.prefs)
```

Comment:
I see what you mean. The switch is about removing giving users the choice from _testing_ rather than removing the feature permanently.  And as long as we arent recommending regular users go to this debug option to disable the feature (which we dont) then I think its fine having the toggle.